docs/developer-docs/smart-contracts/advanced-features/http-gateways.mdx

```diff
@@ -0,0 +1,81 @@
+---
+keywords: [advanced, tutorial, http, http gateways, gateways]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# HTTP gateways
+
+<MarkdownChipRow labels={["Advanced", "Tutorial"]} />
+
+## Overview
+
+The ICP HTTP gateway protocol enables conventional HTTP clients to interact with the ICP network. HTTP gateways run adjacent to ICP and provide a connection used by software such as web browsers send and receive standard HTTP requests and responses, including static assets, such as HTML, JavaScript, images, or videos. The HTTP gateway makes this workflow possible by translating standard HTTP requests into API canister calls that ICP canisters can understand and vice versa for HTTP responses.
+
+There are several HTTP gateway instances maintained for ICP, but it should be noted that an HTTP gateway is a centralized component that could become compromised, creating a threat for users receiving HTTP content. Compromised gateways are threats even for canisters that are considered 'secure', such that they use certify assets, use a default client, and do not maintain their own certification checking.
```

Comment:
This last sentence is the same as before, no? I still find it confusing... I also still think it's misleading to say that gateways are a threat to canisters; they can't influence how the canister behaves. They are a threat to the user though, as they can misrepresent the canister state, potentially forge user commands etc.

Reply:
Not sure what happened to the commit to remove this sentence, but I've re-commited it. This sentence should be removed now.